lint/testdata/indexOnlyLoop/negative_tests.go

```diff
@@ -0,0 +1,27 @@
+package checker_test
+
+import (
+	"fmt"
+	"os"
+)
+
+func printFilesIndexes(files []*os.File) {
+	for i := range files {
+		fmt.Println(i)
+	}
+}
+
+func closeNonPtrFiles(files []os.File) {
+	for i := range files {
+		fmt.Println(files[i].Name())
+		files[i].Close()
+	}
+}
+
+func indexReuse(filesA []*os.File, filesB []*os.File) {
+	for i := range filesA {
+		if filesA[i] == filesB[i] {
+			fmt.Println("equal")
```

Comment:
We generally avoid unnecessary imports in the tests because they cause significant slowdown (until we reuse loader program between tests).
For example, 0 imports => 0.0s test time, 1-2 imports => 1s test time.
This effects sums up.
See also #352.
This is not requirement, just a reminder.

Reply:
removed imports